nil-docs/zkllvm/getting-started/installation.mdx

```diff
@@ -0,0 +1,266 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Installation and configuration
+
+## Using pre-built binaries
+
+The easiest way to access zkLLVM is to install pre-built binaries of its key components. 
+
+:::info
+
+All pre-built binaries discussed below are only supported on x86-x64 systems.
+
+:::
+
+### Install zkLLVM binaries
+
+zkLLVM is distributed as a Deb package which can be installed using the following commands.
+
+```bash
+echo 'deb [trusted=yes]  http://deb.nil.foundation/ubuntu/ all main' >>/etc/apt/sources.list
+apt update
+apt install -y zkllvm
+```
```

Comment:
Maybe we should verify installation here just like with Rust:

```
clang-zkllvm --version
```

It would at least tell user that Clang binary is called `clang-zkllvm`, not `clang`.

Reply:
another great idea, added